BUILDING_LLVM.md

```diff
@@ -0,0 +1,51 @@
+# Building Apple LLVM for Kotlin/Native
+
+This document describes how to compile LLVM distribution and use it to build Kotlin/Native on macOS.  
+Usually, you don't need to compile LLVM by yourself:
+* If you use Kotlin/Native compiler it will download LLVM on the first run. 
+* If you work on Kotlin/Native source code then use `dependency:update` Gradle task.
```

Comment:
Probably `:dependencies:update`.
Also "If you work on Kotlin/Native source code"  -> "If you contribute to kotlin-native repository" (might be more clear).